README.md

```diff
@@ -8,77 +8,103 @@ npm install hypercore-protocol
 
 [![build status](https://travis-ci.org/mafintosh/hypercore-protocol.svg?branch=master)](https://travis-ci.org/mafintosh/hypercore-protocol)
 
+For detailed info on the messages sent on each channel see [simple-hypercore-protocol](https://github.com/mafintosh/simple-hypercore-protocol)
+
 ## Usage
 
 ``` js
-var protocol = require('hypercore-protocol')
+const Protocol = require('hypercore-protocol')
 
 // create two streams with hypercore protocol
-var streamA = protocol({ id: 'a' })
-var streamB = protocol({ id: 'b' })
+const streamA = new Protocol(true) // true because this is the initiator
+const streamB = new Protocol(false) // false because this is not the initiator
```

Comment:
super nitpicky but `true indicates this is the initiator` might be more clear (indicates instead of because)